scripts/core/superdesk-api.d.ts

```diff
@@ -2160,6 +2160,8 @@ declare module 'superdesk-api' {
         previewComponent: React.ComponentType<IPreviewComponentProps>;
         configComponent?: React.ComponentType<IConfigComponentProps<IConfig>>;
         templateEditorComponent?: React.ComponentType<ITemplateEditorComponentProps<IConfig>>;
+
+        onTemplateCreate?(): any;
```

Comment:
```TypeScript
/**
 * Takes field value as an argument and returns modified or same value.
 */
onTemplateCreate?(value: any, config: IConfig): any;
```

Can you use the following interface? It should function in a similar way as `entities.article.onPatchBefore`